api/src/test/java/org/pmiops/workbench/api/CohortsControllerTest.java

```diff
@@ -861,4 +861,24 @@ public void testMaterializeCohortWithEverything() throws Exception {
                 .getBody())
         .isEqualTo(response);
   }
+
+  public static DbConcept makeConcept(Concept concept) {
+    DbConcept result = new DbConcept();
+    result.setConceptId(concept.getConceptId());
```

Comment:
can use mapper here?

Reply:
So it turns out that the tests don't need this method, so I'm removing it.